src/main/scala/org/broadinstitute/dsde/workbench/avram/config/AvramConfig.scala

```diff
@@ -0,0 +1,5 @@
+package org.broadinstitute.dsde.workbench.avram.config
+
+case class AvramConfig(serviceVersion: String,
```

Comment:
It'd be cool to expose the service version in some /version API endpoint. However I wonder if we could set the version in the SBT build file instead of `avram.conf`.

Reply:
👍